excise/tasks.py

```diff
@@ -51,49 +51,65 @@ def api_post_request_task(
         span.set_tag(opentracing.tags.COMPONENT, "tax")
         span.set_tag("service.name", "avatax_excise")
 
-        response = api_post_request(transaction_url, data, config)
-    msg = f"Order sent to Avatax Excise. Order ID: {order.token}"
-    if not response or "Error" in response.get("Status"):
-        errors = response.get("TransactionErrors", [])
-        avatax_msg = ""
-        for error in errors:
-            avatax_msg += error.get("ErrorMessage", "")
-        msg = f"Unable to send order to Avatax Excise. {avatax_msg}"
+        tax_response = api_post_request(transaction_url, data, config)
+
+    msg = ""
+    if not tax_response:
+        msg = (
+            "Empty response received from Excise API, "
```

Comment:
Can you convert it to use `%s` instead of format?  or do something similar to `if 67-82` (separate msg for logger) . Error handlers like Sentry treat such message as a normal string but we can easily have extracted token.

Reply:
Thanks! Fixed here: https://github.com/saleor/saleor-plugin-avatax-excise/pull/11/commits/cbd47fdc24fc59eb16a7eb680a5ceab94fa49641